nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -65,6 +70,18 @@ in
         '';
       };
 
+      user = mkOption {
+        type = types.str;
+        default = "syncserver";
+        description = "User account under which syncserver runs.";
+      };
```

Comment:
Is there a reason you want to make this configurable?

I don't know enough about the firefox sync server but I prefer to make the user not configurable as it makes things simpler.

In my opinion it only makes sense to be able to change the user if the user needs to access data that belongs to the desktop user that has a name that isn't known in advance for example.